example/package.json

```diff
@@ -71,7 +73,6 @@
     "@cumulus/test-processing": "18.2.0"
   },
   "devDependencies": {
-    "@aws-sdk/client-ecs": "^3.447.0",
```

Comment:
Not sure why this was a dev dependency but I think it needs to be a prod dependency.